archunit-junit/junit4/src/main/java/com/tngtech/archunit/junit/internal/ArchTestExecution.java

```diff
@@ -42,6 +47,21 @@ public String toString() {
         return describeSelf().toString();
     }
 
+    Class<?> getTestClass() {
+        return testClassBranch.get(0);
+    }
+
+    String formatWithPath(String testName) {
+        if (testClassBranch.size() <= 1) {
```

Comment:
I didn't pick `testClassPath`, because it reminded me a lot of "test classpath" 🤣 (I had actually thought about it, then pondered more and arrived at "test class branch" 🤪 ) But maybe you're right and the context is clear enough so that "test class path" is clear (since it also came to my mind and your's, must mean it's somehow fitting 😁 )

Reply:
"PathOfTestClasses" 😜

Reply:
Too late 😉 We won't die 😁